source/tests/llassetgen-tests/Image.cpp

```diff
@@ -0,0 +1,129 @@
+#include <ft2build.h>  // NOLINT include order required by freetype
+#include FT_FREETYPE_H
+
+#include <gmock/gmock.h>
+#include <llassetgen/llassetgen.h>
+
+#include <fstream>
+
+using namespace llassetgen;
+
+// current working directory: openll-asset-generator/build/source/tests
+std::string test_source_path = "../../../source/tests/llassetgen-tests/testfiles/";
+std::string test_destination_path = "../../";
+
+TEST(ImageTest, LoadTTF) {
+    init();
+
+    FT_Face face;
+    std::string font_file = test_source_path + "SourceSansPro-Regular.ttf";
+    char letter = 'J';
+
+    FT_Error face_created = FT_New_Face(freetype, font_file.c_str(), 0, &face);
+    EXPECT_EQ(face_created, 0);
+
+    FT_Error pixel_size_set = FT_Set_Pixel_Sizes(face, 64, 64);
+    EXPECT_EQ(pixel_size_set, 0);
+
+    FT_Error glyph_loaded = FT_Load_Glyph(face, FT_Get_Char_Index(face, letter), FT_LOAD_RENDER | FT_LOAD_MONOCHROME);
+    EXPECT_EQ(glyph_loaded, 0);
```

Comment:
If the only thing the test is doing should the condition not be met is crashing then these should be `ASSERT_EQ`s